opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -410,10 +410,25 @@ def get_laser_envelope( self, t=None, iteration=None, pol=None, m='all',
            Only used for thetaMode geometry
            The angle of the plane of observation, with respect to the x axis
 
-        slicing_dir : str, optional
-           Only used for 3dcartesian geometry
-           The direction along which to slice the data
-           Either 'x', 'y'
+        slicing : float or list of float, optional
+           Number(s) between -1 and 1 that indicate where to slice the data,
+           along the directions in `slicing_dir`
+           -1 : lower edge of the simulation box
+           0 : middle of the simulation box
+           1 : upper edge of the simulation box
+           Default is None
+
+        slicing_dir : str or list of str, optional
+           Direction(s) along which to slice the data
+           + In cartesian geometry, elements can be:
+               - 1d: 'z'
+               - 2d: 'x' and/or 'z'
+               - 3d: 'x' and/or 'y' and/or 'z'
+               - 1d/circ: not implemented
+           + In cylindrical geometry, elements can be 'r' and/or 'z'
```

Comment:
The above line seems to contradict `1d/circ: not implemented`. Did I misunderstand?